config/routes.rb

```diff
@@ -60,13 +60,6 @@ def external_redirect(path_prefix, target)
   scope Whitehall.router_prefix, shallow_path: Whitehall.router_prefix do
     root to: redirect("/", prefix: ""), via: :get, as: :main_root
 
-    resources :organisations, only: [] do
-      # These aren't rendered but are coupled to Worldwide organisation corporate information pages
```

Comment:
This comment says they are coupled, but I think it makes sense to start removing code where possible now that we are due to remove the worldwide CIPs too